R/merge_SE.R

```diff
@@ -7,6 +7,10 @@
 #' @param discard_keys character vector of string that will be discarded
 #' during creating BumpyMatrix object
 #'
+#' @examples
+#' se1 <- get_synthetic_data("finalMAE_small.RDS")[[1]]
+#' merge_SE(list(se1 = se1, se2 = se1))
```

Comment:
I get warnings like that: `In .clean_key_inputs(nested_keys, colnames(df_)) :  ignoring input keys: 'fit_source, Metrics_rownames' which are not present in data.table`. Could you confirm or deny?

Reply:
It is expected